example_dags/example_bigquery_dynamic_map_task.py

```diff
@@ -0,0 +1,102 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.decorators import task
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryInsertJobOperator,
+)
+
```

Comment:
Do we need to use all these operators? Would it be an option to promote using existing Astro SDK operators/decorators?

Reply:
Now I'm using aql.loads it would not require